.github/workflows/ci.yml

```diff
@@ -107,12 +105,51 @@ jobs:
       with:
         name: macOS-native-library
         path: bin
-    - name: Build & Run .NET unit tests
-      run: dotnet test csharp.test --configuration=Release
     - name: Build NuGet package
-      run: dotnet pack csharp --configuration=Release
+      run: |
+        dotnet build csharp --configuration=Release
```

Comment:
According to the doc, `dotnet pack` implies `dotnet build`.

Reply:
I started with only `dotnet pack` and got an error about missing `ParquetSharp.dll`, which is why I added `dotnet build`.

After your comment, I just checked with .NET Core 3.1.201 on macOS, and it seems that it's the opposite: `dotnet build` builds the assembly and creates the NuGet package, whilst `dotnet pack` does not build the assembly.

Do you agree to replace the above with only `dotnet build`?

Reply:
You're right. Dotnet seems to have trouble with the native component when doing `dotnet pack`, despite the documentation clearly stating that it should do a build.

Fine, let's go for `dotnet build`, which seems to work for me locally as well.

Reply:
Alright, will change that in the next commit.